testsuite/cluster-test/src/experiments/mod.rs

```diff
@@ -131,7 +140,46 @@ pub fn get_experiment(name: &str, args: &[String], cluster: &Cluster) -> Box<dyn
     );
     known_experiments.insert("generate_cpu_flamegraph", f::<CpuFlamegraphParams>());
     known_experiments.insert("versioning_testing", f::<ValidatorVersioningParams>());
+    known_experiments.insert("compatibility_test", f::<ComaptiblityTestParams>());
 
     let builder = known_experiments.get(name).expect("Experiment not found");
     builder(args, cluster)
 }
+
+/// Reboot `updated_instance` with newer image tag
+pub async fn update_batch_instance(
```

Comment:
I know it is draft PR but have some early ask - I would keep mod.rs for interfaces, can we just move it to CompatibilityTest file and import in versioning test?